rpcs3/main.cpp

```diff
@@ -423,6 +423,8 @@ void run_platform_sanity_checks()
 
 int main(int argc, char** argv)
 {
+	std::setlocale(LC_ALL, "C");
```

Comment:
It may be more future proof to initialize it before main:
e.g.:

static const bool s_init_locale = []()
{
     std::setlocale(LC_ALL, "C");
    return true;
}();